tools/get-contributor-list.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+. "$BASE/tools/includes/check-osx-bash-version.sh"
+. "$BASE/tools/includes/chalk-lite.sh"
+
+# This script obtains the contributor list by comparing the specified version with the previous major version.
+
+function usage {
+	cat <<-EOH
+		usage: $0 [project] <version>
+
+		Obtains the list of contributors for a major release.
+
+	EOH
+	exit 1
+}
+
+if [[ -z $1 ]]; then
+    usage
+    exit 1
```

Comment:
This doesn't hurt, but it'll never be reached since `usage` itself exits on line 18.